res/css/views/settings/tabs/user/_SecurityUserSettingsTab.scss

```diff
@@ -27,14 +27,16 @@ limitations under the License.
 }
 
 .mx_SecurityUserSettingsTab {
-    .mx_SettingsTab_heading {
-        margin-bottom: 22px;
-    }
     .mx_SettingsTab_section {
         .mx_AccessibleButton_kind_link {
             padding: 0;
             font-size: inherit;
         }
+
+        .mx_SettingsTab_subsectionText--top {
+            color: $primary-content; // Same as mx_SettingsTab
```

Comment:
Not sure if we should remove the colour here. Then it would be in sync with all descriptions in this view.

Bonus: We do then won't need `mx_SettingsTab_subsectionText--top` at all.

Reply:
I would like to agree with you but it looks to me it fairly changes the UI impression, so I tried to keep the current UI. Let me take screenshots to compare the UI with and w/o this block.

Reply:
@weeman1337 Please see above.